aries-site/src/pages/foundation/date-and-time.mdx

```diff
@@ -0,0 +1,272 @@
+    }}
+  >
+    <TimezoneExample bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+### ISO 8601 Formatting
+
+ISO 8601 is an internationally recognized standard for presenting date and time. It is presented as `YYYY-MM-DDTHH:mm:ss.sssZ`. For example, `2023-02-07T06:57:00.000Z`.
+
+Some use cases include:
+
+- Log files
+- Trigger times
+
+Even though ISO 8601 is an internationalized standard, it is not the most friendly format for users. In most other cases, date and time formatting should be based on the user's [locale](#locale-formatting) and [timezone](#timezones).
+
+## Relative time
+
+Relative time stamps display:
+
+1. How recently an event has occurred.
+1. How far in the future an event will occur.
+
+Relative time stamps remove the need for the user to perform mental calculations by presenting timestamps in written form like "2 days ago".
+
+When displaying relative time:
+
+- Adapt and show the appropriate unit of time. For example, after 60 minutes, display hours.
+- Unless required, displaying the time for anything less than a second and more than a week is generally not needed.
+- For older events, such as anything that happened more than a week ago, consider using [absolute time](#absolute-time) instead.
+- Consider whether or not it's important for users to be able to see the absolute time as well. This could be placed within a tooltip or displayed on a detail page.
+
+| Description            | Display                          | Display when space is limited   |
+| ---------------------- | -------------------------------- | ------------------------------- |
+| Within the last minute | Just now; a minute ago           | now; a min ago                  |
+| 1 day ago              | yesterday                        | yesterday                       |
+| < 1 week ago           | x days, hour, minute, second ago | x days, hr, min, sec ago        |
+| >= 1 week ago          | [Absolute time](#absolute-time)  | [Absolute time](#absolute-time) |
+| In 1 day               | tomorrow                         | tomorrow                        |
+| < 1 week from now      | in x days, hour, minute, second  | in x days, hr, min sec          |
+| In >= 1 week           | [Absolute time](#absolute-time)  | [Absolute time](#absolute-time) |
+
+<BestPracticeGroup>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'do',
+      message:
+        'Provide users with only relevant information. In this use case, users do not need to know how many seconds ago an activity was last performed or the exact time activities older than a month were performed.',
+    }}
+  >
+    <RelativeTimeExample />
+  </Example>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'dont',
+      message: `Don't include additional information if it isn't needed. This will hurt readability and cause issues with layouts at smaller breakpoints.`,
```

Comment:
Originally I had this as right aligned but I know we haven't defined that quite yet. Just flagging for future consideration if we want to realign this example.

Reply:
I've right aligned this based on discussion to add the alignment section.